release/BUILD.bazel

```diff
@@ -75,7 +75,7 @@ genrule(
     srcs = [
         ":config.yaml.tmpl",
         ":install.sh",
-        "//:component-version",
+        "//:VERSION",
```

Comment:
I think this change might break things as the latest version will probably be the latest version that has the +100 bump.

Reply:
The previous versions have "nightly-" which should make them get filtered out by the assistant's logic. Do you think this will mess things up with bintray? It's ok for the bintray version to still be +100 (we account for that in the URL logic above). It's also ok IMO for bintray downloads to break since we're moving to github releases with @garyverhaegen-da's help.

Reply:
I am slightly worried that the `latest` URL will resolve to the `+100` version. If you are confident that this will not be a problem that’s fine :+1:

Reply:
It should resolve to the +100 version! I guess we'll find out if it breaks bintray downloads ... oh well.